### DISCUSSION TITLE
Jyer's Firmware check boxes don't look right

### DISCUSSION THREAD

**Discussion by allenj444** [Jyers/Marlin#2057] [General Discussion]
Hello, has anyone else had funky looking check boxes on the Jyer's firmware display? (photo attached)

![IMG_6515](https://user-images.githubusercontent.com/117777969/200675078-0e90c667-cffc-46d5-90a5-f756679f90bf.JPG)

**Comment by mickiete** [comment]
Yes, I have the same.
